### ISSUE TITLE
Changing the learning rate affect the accuracy

### ISSUE THREAD

**Issue by CynthiaProtector** [apache/incubator-mxnet#14917]
## Description
During the training process, I tried to change the learning rate to 0.0 after 200 steps, and I found that the accuracy experiences a dropoff. Why is that? 

The following figure shows the training process of Lenet with the MNIST dataset. I use one server and one worker (two different nodes), and the worker contains 2 GPUs. I start the training task with the following command:

`python ../../tools/launch.py -s 1 -n 1 --launcher ssh -H hosts python train_mnist.py --network lenet --kv-store dist_sync --gpus 0,1`

![lr-200change](https://user-images.githubusercontent.com/23467320/57387914-cb154c00-71e9-11e9-86cc-b4ca2458359f.png)

**Comment by vrakesh** [comment]
@CynthiaProtector Thank you for , posting your query, this is a interesting find, not sure if this is expected behavior, will look into this

@mxnet-label-bot add [Question]

**Comment by vrakesh** [comment]
Hi @CynthiaProtector turns out this is expected behavior,  once learning rate is set to zero , the training essentially starts, running training on the next batch of data with learning rate set to zero basically means you are being shown test accuracy rather than training accuracy.

I would suggest a non zero small amount of decay instead, to see if that helps.

But setting learning rate to zero essentially means stopping gradient descent, no further improvements on weights are made on new batches of data

If this answer , satisfies your query, Feel free to close this issue

**Comment by vrakesh** [comment]
@mxnet-label-bot add [Pending Requester Info]

**Comment by CynthiaProtector** [comment]
Thank you for your explanation @vrakesh and I feel sorry to response a little late. The explanation seems to be reasonable and I will conduct some more experiments to check the mechanism. The issue is closed and once I have some new findings, I will reopen this issue.
